src/Evolution/Systems/Cce/ReadBoundaryDataH5.cpp

```diff
@@ -262,7 +262,8 @@ ReducedSpecWorldtubeH5BufferUpdater::ReducedSpecWorldtubeH5BufferUpdater(
       Spectral::Swsh::Tags::SwshTransform<Tags::Dr<Tags::BondiJ>>>>(
       dataset_names_) = "DrJ";
   get<Tags::detail::InputDataSet<
-      Spectral::Swsh::Tags::SwshTransform<Tags::BondiH>>>(dataset_names_) = "H";
+      Spectral::Swsh::Tags::SwshTransform<Tags::Du<Tags::BondiJ>>>>(
+      dataset_names_) = "H";
```

Comment:
should the label also change?

Reply:
It shouldn't, but the reasoning is a little subtle: because H is coordinate dependent, so what's in the file is H in bondi coordinates (which makes sense for what is stored), but what we call H in the code is the H in numerical coordinates, because that is what is needed for the time stepping.